.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: Release
+
+on: [push]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
I guess we would need another token to work with branch protection?
https://intuit.github.io/auto/docs/build-platforms/github-actions#running-with-branch-protection

Reply:
Hmm... I thought I was handling that here: https://github.com/grafana/plugin-tools/pull/127/files/d97b07a16ec94d275c069eecf5033977bbf7f965#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R38

Reply:
Is it enough to only have this token for release step only? If yes we can leave it be as is.